dadi/lib/controller/index.js

```diff
@@ -213,39 +213,37 @@ Controller.prototype.post = function (req, res, next) {
   debug('POST %s %o', pathname, req.params)
 
   // flush cache for POST requests
-  help.clearCache(pathname, (err) => {
-    if (err) return next(err)
-
-    // if id is present in the url, then this is an update
-    if (req.params.id || req.body.update) {
-      internals._lastModifiedAt = Date.now()
-      internals._lastModifiedBy = req.client && req.client.clientId
-
-      var query = {}
-      var update = {}
-
-      if (req.params.id) {
-        query._id = req.params.id
-        update = req.body
-      } else {
-        query = req.body.query
-        update = req.body.update
-      }
-
-      // add the apiVersion filter
-      if (config.get('query.useVersionFilter')) {
-        query._apiVersion = internals._apiVersion
-      }
+  help.clearCache(pathname, () => {})
```

Comment:
Just a minor detail: I'd rather change the `helpCache` implementation to make the callback argument optional than having to pass an empty function to it.